scripts/build_ios.sh

```diff
@@ -0,0 +1,70 @@
+#!/bin/bash
+
+PROJECT_DIR=$(pwd)
+
+# Check for the device type argument
+if [[ "$1" == "simulator" ]]; then
+    DEVICE_TYPE="simulator"
+    ARCHITECTURE="aarch64-apple-ios-sim"
+elif [[ "$1" == "device" ]]; then
+    DEVICE_TYPE="device"
+    ARCHITECTURE="aarch64-apple-ios"
+else
+    echo -e "${RED}Error: Please specify either 'simulator' or 'device' as the first argument.${DEFAULT}"
+    exit 1
+fi
+
+# Check for the build mode argument
+if [[ "$2" == "debug" ]]; then
+    BUILD_MODE="debug"
+    LIB_DIR="debug"
+elif [[ "$2" == "release" ]]; then
+    BUILD_MODE="release"
+    LIB_DIR="release"
+else
+    echo -e "${RED}Error: Please specify either 'debug' or 'release' as the second argument.${DEFAULT}"
+    exit 1
+fi
+
+# build circom circuits in mopro-core
+
+cd ${PROJECT_DIR}/mopro-core/examples/circom/keccak256
+npm install
```

Comment:
Do we want to build and prepare circuits everytime? I think this can probably be a separate `prepare` script